notify.js

```diff
@@ -10,24 +10,32 @@
     if (typeof define === 'function' && define.amd) {
         // AMD environment
         define(function() {
-            return factory(global, global.document);
+            return factory(global, global.document, global.navigator);
         });
     } else if (typeof module !== 'undefined' && module.exports) {
         // CommonJS environment
-        module.exports = factory(global, global.document);
+        module.exports = factory(global, global.document, global.navigator);
     } else {
         // Browser environment
-        global.Notify = factory(global, global.document);
+        global.Notify = factory(global, global.document, global.navigator);
     }
-} (typeof window !== 'undefined' ? window : this, function (w, d) {
+} (typeof window !== 'undefined' ? window : this, function (w, d, n) {
     'use strict';
 
     var N = w.Notification;
+    var SW = n.serviceWorker;
 
     function isFunction(item) {
         return typeof item === 'function';
     }
 
+    function bindNotificationEvents(notify) {
```

Comment:
I'd probably favor adding this to the `prototype` if it's going to be a function?